policy/README.md

```diff
@@ -31,3 +31,92 @@ The following policy agent configuration directives are currently supported:
 ### `opa` backend configuration
 
 Currently, `opa` backend does not support any configuration.
+
+## Policy Identification
+
+There are three different ways of identifying a policy:
+
+### appraisal policy ID
+
+This is an identifier used in the attestation result as defined by [EAR
+RFC](https://www.rfc-editor.org/rfc/internet-drafts/draft-fv-rats-ear-00.html#name-ear-appraisal-claims).
+Note that in this case, "policy" is used in the sense of the [RATS
+architecture](https://datatracker.ietf.org/doc/html/draft-ietf-rats-architecture-05)
+policy. In Veraison, this encompasses both, the scheme, and the policy applied
+from the policy store by a policy engine.
+
+An appraisal policy ID is a [URI](https://www.rfc-editor.org/rfc/rfc3986) with
+the scheme `policy` followed by a rootless path indicating the (RATS) policy
+using which the appraisal has been generated. The first segment of the path is
+the name of the scheme used to create the appraisal. The second segment, if
+present, is the individual policy ID (see below) of the policy that has been
+applied to the appraisal created by the scheme.
+
+For example:
+
+- `policy:TPM_ENACTTRUST`: the appraisal has been created using "TPM_ENACTTRUST" scheme, with
+  no additional policy applied.
+- `policy:PSA_IOT/0:opa:v1`: the appraisal has been created using "PSA_IOT"
```

Comment:
For cascading policies the tenant ID is repeated in every segment, so it's worth factoring out into the first segment.  Especially considering the fact that tenant IDs may not be that short (e.g., UUIDs or other kinds of long-ish identifiers).

So something like: `policy:0:PSA_IOT/opa:v1`

Reply:
This is now obsolite -- the individual policy ID (here in the format `[tenant]:[name]:[version]` will be replaced with the policy instances' UUID instead (as be discussion last Thursday).